blocks/results-list-block/features/results-list/results/result-item.jsx

```diff
@@ -1,86 +1,151 @@
 import React from "react";
+import { useEditableContent } from "fusion:content";
+import { useComponentContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import { RESIZER_APP_VERSION, RESIZER_URL } from "fusion:environment";
+import { localizeDateTime } from "@wpmedia/engine-theme-sdk";
 
-import ArticleDate from "@wpmedia/date-block";
-import { Image } from "@wpmedia/engine-theme-sdk";
-import { extractResizedParams, extractImageFromStory } from "@wpmedia/resizer-image-block";
-import { Byline, Heading, SecondaryFont, Overline } from "@wpmedia/shared-styles";
+import {
+	Attribution,
+	Separator,
+	Heading,
+	Overline,
+	Image,
+	Link,
+	Date as DateComponent,
+	formatAuthors,
+	Paragraph,
+	Join,
+	formatURL,
+	MediaItem,
+	Conditional,
+	getImageFromANS,
+	imageANSToImageSrc,
+	usePhrases,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-results-list";
 
 const ResultItem = React.memo(
 	React.forwardRef(
 		(
 			{
 				arcSite,
 				element,
-				imageProperties,
 				targetFallbackImage,
-				placeholderResizedImageOptions,
 				showByline,
 				showDate,
 				showDescription,
 				showHeadline,
 				showImage,
 				showItemOverline,
+				overlineURL,
```

Comment:
The `overlineURL` should be derived from the ANS content

Reply:
missed it. Working on this

Reply:
Fixed it